extensions/stackdriver/metric/registry.cc

```diff
@@ -17,15 +17,17 @@
 #include "extensions/stackdriver/common/constants.h"
 #include "google/api/monitored_resource.pb.h"
 
+#include <typeinfo>
```

Comment:
nits: IIRC bracket include header should be next to the top most "registry.h" 

Feel free to ignore if it's not.

Reply:
Also: where is it used?

Reply:
Oops, yeah, this is left from debugging. Will clean up in following PR. Thanks!